gulpfile.js

```diff
@@ -205,12 +205,14 @@ function install(done) {
                     .pipe(gulp.dest(config.dist.assets + key + '/decoders'));
                 gulp.src('node_modules/' + key + '/locales/**/*')
                     .pipe(gulp.dest(config.dist.assets + key + '/locales'));
-            } else if (key == 'bootstrap' || key == 'bootstrap-rtl' || key == 'bootswatch') {
-                // bootstrap, bootstrap-v4-rtl, and bootswatch are special and need to copy dist and scss
+            } else if (key == 'bootstrap' || key == 'bootstrap-rtl' || key == 'bootswatch' || key == '@fortawesome/fontawesome-free') {
+                // bootstrap, bootstrap-v4-rtl, bootswatch, and fontawesome are special and need to copy dist and scss
                 gulp.src('node_modules/' + key + '/dist/**/*')
                     .pipe(gulp.dest(config.dist.assets + key + '/dist'));
                 gulp.src('node_modules/' + key + '/scss/**/*')
                     .pipe(gulp.dest(config.dist.assets + key + '/scss'));
+                gulp.src('node_modules/' + key + '/webfonts/**/*')
+                    .pipe(gulp.dest(config.dist.assets + key + '/webfonts'));
```

Comment:
@tywrenn , to clarify, this webfonts is only needed for fontawesome, correct?
(if so, then I'll put a quick if block there; not really for functionality just in case we decide to reorg the code in future and will be clear what dirs are needed for each package)